narwhals/_dask/expr.py

```diff
@@ -74,11 +74,9 @@ def _from_call(
         def func(df: DaskLazyFrame) -> list[Any]:
             results = []
             inputs = self._call(df)
+            _args = [maybe_evaluate(df, x) for x in args]
+            _kwargs = {key: maybe_evaluate(df, value) for key, value in kwargs.items()}
             for _input in inputs:
-                _args = [maybe_evaluate(df, x) for x in args]
-                _kwargs = {
-                    key: maybe_evaluate(df, value) for key, value in kwargs.items()
-                }
```

Comment:
`maybe_evaluate`-int is loop independent

Reply:
nice one!